spec/services/target/create_rspec.rb

```diff
@@ -92,5 +92,21 @@
         expect(response).to have_http_status(:unauthorized)
       end
     end
+
+    context 'when the user has 9 targets' do
+      let(:targets) { create_list(:target, 9, user: user) }
+      it 'the target create correct' do
+        subject
+        expect(response).to have_http_status(:success)
```

Comment:
you will not have a response here.. your subject should be `CreateService.new(user, params).call` and it will return you a Target